CHARTER.md

```diff
@@ -0,0 +1,101 @@
+# DevRel Foundation Structure and Organization Working Group Operating Charter (DRAFT)
+
+## 1. Working Group Definition
+
+### Tl;Dr:
+The primary objective of this working group is to document best practices for structuring and organizing Developer Relations (DevRel) teams within companies, including:
+- Team size
+- Roles and responsibilities
+- Reporting structure
```

Comment:
may be worth considering adding how interactions with other roles is considered as well, not just reporting structure but cross-functional collaboration